movai_core_shared/core/message_client.py

```diff
@@ -105,10 +140,17 @@ def foraward_request(self, request_msg: dict) -> dict:
         Args:
             request_msg (dict): The request to forward.
         """
-        request = {"request": request_msg}
+        if "request" not in request_msg:
+            request = {"request": request_msg}
         self._zmq_client.send(request)
-        if request_msg["response_required"]:
-            return self._zmq_client.recieve()
+        response_required = request_msg.get("response_required")
+
+        if response_required is None:
+            response_required = False
```

Comment:
no need in this line
if None -> false

and you can do in the previous line
get(response_requird, None)

Reply:
done